.github/workflows/e2e_test_ci.yml

```diff
@@ -1027,3 +1027,107 @@ jobs:
           run: |
             killall spice || true
             cat spice.log
+
+  mssql_connector_quickstart:
+      name: 'Test MS SQL connector on ${{ matrix.target.name }}'
+      runs-on: ${{ matrix.target.runner }}
+      needs:
+        - build
+        - setup-matrix
+
+      strategy:
+          matrix:
+            target: ${{ fromJson(needs.setup-matrix.outputs.matrix) }}
+            # run only on "Linux x64"
+            exclude:
+              - target:
+                  target_os: "darwin"
+
+      services:
+        sqlserver:
+          image: mcr.microsoft.com/mssql/server:2019-latest
```

Comment:
If you use `"slyons/adventureworks:latest-2019"` it's preloaded with the AdventureWorks dataset. Here's what I'm using in my branch right now:

```yaml
mssql1:
    image: "slyons/adventureworks:latest-2019"
    environment:
      - ACCEPT_EULA=Y
      - SA_PASSWORD=Password123@
    restart: unless-stopped
    ports:
      - "11433:1433"
    healthcheck:
      test: [ "CMD", "./ready.sh"]
      interval: 5s
      timeout: 5s
      retries: 5
```